controllers/parcels.js

```diff
@@ -1,25 +1,29 @@
-const uuid = require('uuid');
-const moment = require('moment');
+import { v4 as uuidv4 } from 'uuid';
+import moment from 'moment';
 
-const {
+import {
   sendParcelValidation,
   receiveParcelValidation,
-} = require('../middlewares/validation');
-const parcels = require('../model/parcels');
+} from '../middlewares/validation.js';
+import parcels from '../model/parcels.js';
 
-exports.parcel_get_all = (req, res) => res.send(parcels);
+export const parcel_get_all = (req, res) => res.send(parcels);
 
-exports.parcel_get_user_history = (req, res) => {
+export const parcel_get_user_history = (req, res) => {
   const sentParcels = parcels.filter(
     (parcel) => parcel.fromUser == req.user.email
   );
-  const receivedParcels = parcels.filter(
-    (parcel) => parcel.toUser == req.user.email
-  );
-  res.json({ sentParcels, receivedParcels });
+  try {
+    const receivedParcels = parcels.filter(
+      (parcel) => parcel.toUser == req.user.email
+    );
+    res.json({ sentParcels, receivedParcels });
+  } catch (err) {
+    res.status(500).send('Something went wrong');
```

Comment:
Return something in try catch
```suggestion
  try {
    const receivedParcels = parcels.filter(
      (parcel) => parcel.toUser == req.user.email
    );
    return res.json({ sentParcels, receivedParcels });
  } catch (err) {
    return res.status(500).send('Something went wrong');
```